WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1434,6 +1434,7 @@ private void setEmptyTitleDescriptionAndButton(boolean requestFailed) {
         int heightToolbar = getActivity().getResources().getDimensionPixelSize(R.dimen.toolbar_height);
         int heightTabs = getActivity().getResources().getDimensionPixelSize(R.dimen.tab_height);
         mActionableEmptyView.updateLayoutForSearch(false, getEmptyViewTopMargin());
+        mActionableEmptyView.subtitle.setContentDescription(null);
```

Comment:
I think the most appropriate location is here where most of the `subtitle` manipulation are happening:

https://github.com/wordpress-mobile/WordPress-Android/blob/ff4c3b3876720a69aafb5ee837d4e38981dcd3d8/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java#L1541-L1560

This line will be more discoverable there.

Reply:
I didn't put it there because the line that's actually setting the `contentDescription` is here